lib/write.js

```diff
@@ -146,14 +146,7 @@ exports.usingStreaming = (deviceFileDescriptor, options = {}) => {
           transferredBytes += chunk.length;
 
           if (chunk.length < options.chunkSize) {
-            const finalChunkSize = _.attempt(() => {
-              if (chunk.length > options.minimumChunkSize) {
-                return options.chunkSize;
-              }
-
-              return options.minimumChunkSize;
-            });
-
+            const finalChunkSize = Math.ceil(chunk.length / options.minimumChunkSize) * options.minimumChunkSize;
```

Comment:
LGTM - I guess this is basically rounding up the `finalChunksize` to the nearest multiple of 512 bytes?